newton-raphson.js

```diff
@@ -4,6 +4,9 @@ define(["parser/parser", "./matrix"], function(Parser, Matrix){
 	var variables = [];
 	var epsilon = 1e-6;
 	var stoppingCriterion = 1e-3;
+	var precision = 12;
+	var n = 1;
+	var m = 0;
```

Comment:
@sachingrover211  Why are n and m declared here?  Are they likely to be re-used anywhere else in the program in the future?  If so there should be documentation here.  In the place where they do get used (getInitialPoint), it seems like they are never modified...I think I must be missing something.

Reply:
I was trying to experiment with the ranges. So quantities that can effect the result are at the top. That is the only idea here.

Reply:
@jonwetzel let me know what do you think about it after the response.